client/src/main/java/io/split/client/SplitClientConfig.java

```diff
@@ -60,6 +66,8 @@ private SplitClientConfig(String endpoint,
         _ready = ready;
         _debugEnabled = debugEnabled;
         _labelsEnabled = labelsEnabled;
+        _impressionListener = impressionListener;
+        _impressionListenerCapacity = impressionListenerCapacity;
```

Comment:
There is no default. The method in the builder is:

public Builder impressionListener(ImpressionListener impressionListener, int capacity) {
  // code
}

which means that when you provide the impression listener, you also provide the capacity